src/lib.rs

```diff
@@ -508,8 +508,8 @@ impl Level {
 
     /// Converts the `Level` to the equivalent `LevelFilter`.
     #[inline]
-    pub fn to_level_filter(&self) -> LevelFilter {
-        LevelFilter::from_usize(*self as usize).unwrap()
+    pub fn to_level_filter(self) -> LevelFilter {
```

Comment:
These are breaking changes.

Reply:
My intention here is to use the enum instead of the reference, which is faster caus it's a small enum, but you're right, it's going to break stuff.
I'm thinking about an Into implementation

Reply:
I've implemented `From<Level> for LevelFilter` so we can convert from Level into LevelFilter not needing to use the reference.